Modules/_ctypes/callproc.c

```diff
@@ -1123,6 +1123,15 @@ PyObject *_ctypes_callproc(PPROC pProc,
                 goto cleanup; /* leaking ? */
             }
         }
+#if (defined(__x86_64__) && (defined(__MINGW64__) || defined(__CYGWIN__))) || \
+        defined(__aarch64__)
+        void *tmp;
+        if (pa->ffi_type->type == FFI_TYPE_STRUCT) {
```

Comment:
I should correct myself again--actually it is needed for smaller structs if they are not register-aligned.